addOns/graphql/src/test/java/org/zaproxy/addon/graphql/VariantGraphQIUnitTest.java

```diff
@@ -127,6 +129,29 @@ void shouldLogWarnOnInvalidJson() throws HttpMalformedHeaderException {
         assertThat(logMessages.size(), is(1));
     }
 
+    @Test
+    void shouldSetGraphqlQueryParametersCorrectlyOnTheUrl()
+            throws HttpMalformedHeaderException, URIException {
+        // Given
+        HttpRequestHeader httpReqHeader = new HttpRequestHeader();
+        httpReqHeader.setMessage(
+                "GET /graphql/?query=%7BsqlInjection(expression:%20%221%22)%7D HTTP/1.1");
```

Comment:
This should just be done via an HttpRequestHeader constructor. 
While I understand it works this way setMessage is meant to be for body content (as far as I can tell).

https://github.com/zaproxy/zaproxy/blob/8780aa5e4af1c6287bbf291a75315f1df621feb5/zap/src/main/java/org/parosproxy/paros/network/HttpRequestHeader.java#L224

Reply:
Thanks @kingthorin. I can change it but I it is how all the other tests on the file are done.

Reply:
Hmmm okay leave it 🤷‍♂️